src-tauri/tauri.conf.json

```
@@ -57,8 +57,8 @@
         "title": "Advanced Cockpit Emulator",
         "decorations": false,
         "fullscreen": false,
-        "height": 600,
-        "width": 800,
+        "height": 800,
+        "width": 1200,
```

I found that this width looks a little less awkward.
```suggestion
        "width": 1100,
```